readme-eng.md

```diff
@@ -1,222 +1,272 @@
-# Introduction
 
 * [日本語](/readme.md)
 
-This readme is very old, so I will update it later.
-
 # How to install
 
-##  UNIX
+## Windows
 
-Download source archive and try...
+ * Download the latest binaries from [Releases](https://github.com/hengband/hengband/releases) - look for a `.zip` archive.
+ * Unpack the archive in the directory of your chosing.
+ * Double click on `Hengband.exe` to start the game.
 
-```
-tar -zxvf hengband-x.x.x.tar.gz
-cd hengband-x.x.x
-./configure --disable-japanese
-make install
-```
+**Note:** The first time you try Windows Smartscreen may ask you if you want to run this software. To bypass it you need
+to first click on the text link *More info* and then the button *Run anyway*
 
-Substitute the version number to x.x.x.
-You need --disable-japanese option of the configure for English version.
-If you are a super user, --with-setgid=games option works.
-then try
+### Menu bar:
 
-```
-./hengband -- -n<number of windows>  ## for normal ASCII graphics
-```
+Some elements of the interface and game can be customized using the menu bar:
 
-  or
+ * **Sound Effects:** Options -> Sound
+ * **Music:** Options -> Music
+ * **Background image:** This will display an image in the background of the game.
+   * Options -> Background image -> None - disable the background
+   * Options -> Background image -> Custom - image selected by you (on first select will prompt you to select a file)
+   * Options -> Background image -> Change custom image - changes the selected Custom image
 
-```
-./hengband -g -- -n<# of windows>    ## for 8x8 tile graphics
-./hengband -g -- -b -n<# of windows> ## for Bigtile graphics
-./hengband -g -- -a -n<# of windows> ## for 16x16 tile (need heng_graf.tar.gz)
-```
+## UNIX
 
-  to play.
-
-You can change game windows' font, location, and size, by environment 
-variables.
-
-Ex.
+Download the latest version of the source files from [Releases](https://github.com/hengband/hengband/releases). Then run the following commands:
 
 ```
-setenv ANGBAND_X11_FONT '-*-*-medium-r-normal--24-*-*-*-*-*-iso8859-1'
-hengband -- -n
+tar -jxvf hengband-x.x.x.tar.bz2
+cd hengband-x.x.x
+./configure
+make install
 ```
 
-Then font size will be changed.
+**Please note:**
 
-You can set ANGBAND_X11_FONT_n for specific window which have window number n.
+ * Replace `x.x.x` with the downloaded version's number.
+ * `--disable-japanese` option is required to build English version.
```

Comment:
I'd recommend putting the --disable-japanese option after ./configure in the commands above (since this is the English readme) and putting a note here to drop the --disable-japanese option for ./configure if one wants to build the Japanese version.  If you'd like to keep it like it is, I'd recommend clarifying that --disable-japanese is for the configure command and not one of the other commands, i.e. "* `--disable-japanese` option to ./configure is required to build the English version".